### PR TITLE
DOC: Clarify centroid subtlety

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to clarify centroid subtlety that it requires background subtraction to be accurate.

Related PRs:

* #1823
* #1841 

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)? [🐱](https://jira.stsci.edu/browse/JDAT-2933)

### CODE REVIEW

File: docs/imviz/plugins.rst [hunk 1]
```diff
@@ -59,6 +59,14 @@ center-of-mass of the data within the aperture.
 No background subtraction is performed. Click :guilabel:`Recenter`
 to change its parameters and move it to the calculated centroid.
 
+.. note::
+
+    If you want accurate centroid calculations, it is recommended that you
+    use a background subtracted image. Alternately, you could calculate
```
Comment (larrybradley) [inline docs/imviz/plugins.rst:65]:
```suggestion
    use a background-subtracted image. Alternately, you could calculate
```

**Review by rosteen — APPROVED** [review]
LGTM!

**Comment by pllim** [comment]
@larrybradley , you happy with the wordings now?

**Review by larrybradley — APPROVED** [review]
LGTM

**Pull request merged**